modules/polling/components/review/ReviewBox.tsx

```diff
@@ -59,7 +63,9 @@ export default function ReviewBox({
 
   const { data: precheckData } = useSWR(
     account && ballotPollIds && ballotPollIds.length > 0 && network
-      ? `/api/polling/precheck?network=${network}&voter=${account}&pollIds=${ballotPollIds.join(',')}`
+      ? `/api/polling/precheck?network=${network}&voter=${account}${
```

Comment:
i think it would be better if you just send the wallet address, and in the api you check if it has any proxy or delegateaddress instead of having to send all the info

Reply:
I was actually going to say I don't think there's any issue just using the wallet address when dealing with the cache check in the precheck and vote api.  The point of that check is to make sure someone can't vote in the same poll 100 times before their vote shows up in spock. It shouldn't matter if that address has a vote proxy or delegate contract.

Reply:
For the cache is fine to use the user's wallet address, but I'm not sure if we can use only that when fetching the voting history. I'll check that.

My initial approach was fetching the proxy contract and delegate contract on the precheck endpoint but it makes the user interaction slower